File: src/Application/LagFinanceApplication/Models/Movimentacoes/MovimentacaoModel.cs
using LagFinanceDomain.Enum;

namespace LagFinanceApplication.Models.Movimentacoes
{
    public class MovimentacaoModel
    {
        public string Descricao { get; set; }

        public string Observacao { get; set; }

        public decimal Valor { get; set; }

        public DateTime Data { get; set; }

        public TipoMovimentacaoEnum TipoMovimentacao { get; set; }

        public bool Pendente { get; set; }

        public string Conta { get; set; }

        public string ContaTransferencia { get; set; }

        public string Categoria { get; set; }
    }
}

File: client/src/finance/movimentacao/models/movimentacao.model.ts
import { TipoMovimentacaoEnum } from "./tipoMovimentacao.model"

export class Movimentacao {
    Id: string = ''
    Descricao: string = ''
    Observacao: string = ''
    Valor: number = 0
    Data: Date = new Date()
    ContaId: string = ''
    CategoriaId: string = ''
    Tipo: TipoMovimentacaoEnum = TipoMovimentacaoEnum.Despesa
    Pendente: boolean = false
} 


